logging/src/main/java/org/jboss/as/logging/handlers/SyslogHandlerResourceDefinition.java

```diff
@@ -118,6 +120,11 @@ public void marshallAsElement(final AttributeDefinition attribute, final ModelNo
             .setValidator(EnumValidator.create(SyslogType.class, EnumSet.allOf(SyslogType.class)))
             .build();
 
+    public static final SimpleAttributeDefinition NAMED_FORMATTER = SimpleAttributeDefinitionBuilder.create(AbstractHandlerDefinition.NAMED_FORMATTER)
+            .setAlternatives()
```

Comment:
Something missing?

Reply:
Probably not; looks like cruft.

Reply:
This was to remove the alternatives as the `syslog-handler` doesn't and won't have a `formatter` attribute. All the other resources that have a `named-formatter` attribute have an alternative set of `formatter`. It probably makes sense to document why this is done though.